src/parser/jobs/ast/index.tsx

```diff
@@ -35,6 +35,7 @@ export const ASTROLOGIAN = new Meta({
 
 	contributors: [
 		// {user: CONTRIBUTORS.YOU, role: ROLES.YOUR_ROLE},
+		{user: CONTRIBUTORS.OTOCEPHALY, role: ROLES.DEVELOPER},
 	],
 
 	changelog,
```

Comment:
This change should have a changelog entry, as it is changing job logic (the Defensives being tracked)

Reply:
@otocephaly Yeah, I agree. Please add a quick changelog(s) to just summarize your new module and what's been changed.

Reply:
Added! (Originally I didn't add it since we are combining the other changes, but this makes sense to separate since it's a new module)

Reply:
@AzariahAeroborn Can you confirm the changes? Thanks!